tests/e2e/basic_test.go

```diff
@@ -193,8 +194,13 @@ func (s *BasicTestSuite) TestVmGroupVolumeIsolation(c *C) {
 	out, err = dockercli.DeleteVolume(s.vm1, s.volName2)
 	c.Assert(err, IsNil, Commentf(out))
 
+	// Clean up the vm group
+	out, err = admincli.DeleteVMgroup(s.esx, vmgroup)
+	c.Assert(err, IsNil, Commentf(out))
+
 	// Remove Config DB
-	admincli.ConfigRemove(s.esx)
+	out, err = admincli.ConfigRemove(s.esx)
```

Comment:
"err check is not doing anything in case of admin cli" - this is true, but this is not the test case issue, this is a product issue, and we already have a tracking issue for that: #1411 

"We should add this check to verify mode is Not configured." - We need an entire test group to verify different state changes: #1456